src/views/IncorporationAgreement.vue

```diff
@@ -0,0 +1,316 @@
+      <header id="share-structure-header">
+        <h2>2. Sample Templates</h2>
+      </header>
+      <p>
+        For your convenience, we have provided a sample Incorporation Agreement and a set of Sample Benefit Company
+        Articles.
+      </p>
+      <div>
+        <v-card flat class="share-structure-check-panel">
+          <div class="share-structure-check-header">
+            <v-icon color="black">mdi-information-outline</v-icon>
+            <span> Share Structure Check:</span>
+          </div>
+          <div class="share-structure-check-text">
+            The sample articles CAN ONLY be used if the company's share structure DOES NOT contain a class or series
+            with special rights or restrictions.
+            <span class="read-more-btn" @click="readMoreFlag = true">
+              <span v-if="!readMoreFlag">Read more...</span>
+            </span>
+            <div v-if="readMoreFlag">
+              <div class="read-more-line">
+                If the corporation's share structure contains a class or series with special rights or restrictions, you
+                need to create a unique set of Articles for the company and outline these special rights or restrictions
+                in the Articles.
+              </div>
+              <div class="read-more-line">
+                We recommend seeking professional assistance from a lawyer or accountant to help you prepare your
+                Incorporation Agreement and Articles.
+                <span class="read-more-btn" @click="readMoreFlag = false">
+                  <span>Read less...</span>
+                </span>
+              </div>
+            </div>
+          </div>
+          <div class="preview-download-container">
+            <div>
+              <img src="@/assets/images/BCRegistries_Sample_IncoporationAgreement_x2.png" class="preview-image" />
+            </div>
+            <div class="download-link-container">
+              <span>
+                <v-icon color="blue">mdi-file-pdf-outline</v-icon>
+                <a href="/files/benefit_company__corporation_agreement.pdf" download>
+                  Download the Sample Incorporation Agreement and Benefit Company Articles
+                </a>
+              </span>
+            </div>
+          </div>
+        </v-card>
+      </div>
+    </section>
+    <section class="mt-10">
+      <header id="share-structure-header">
+        <h2>3. Confirm Incorporation Agreement and Article Completion</h2>
+      </header>
+      <AgreementType />
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import { AgreementType } from '@/components/IncorporationAgreement'
+
+@Component({
+  components: {
+    AgreementType
+  }
+})
+export default class IncorporationAgreement extends Vue {
+  private helpToggle: boolean = false
+  private readMoreFlag: boolean = false
+
+  private get documentURL (): string {
```

Comment:
Is this used anywhere?

Reply:
No, as I mentioned I put it there for reviewers to reproduce the scenario I mentioned about document download.  I will be removing that